src/main/resources/application-localstack.yml

```diff
@@ -0,0 +1,65 @@
+#
+# The "-local" profile is for running the API service locally against pre-existing
+# docker containers that are listening on a known port numbers for a postgres database (5432).
+#
+
+# Most values are inherited from - application.yml - except the following:
+spring:
+  jpa:
+    show-sql: true
+
+  flyway:
+    locations: classpath:/migrations/common,classpath:/migrations/dev
+
+system:
+  client:
+    id: ${SYSTEM_CLIENT_ID}
+    secret: ${SYSTEM_CLIENT_SECRET}
+
+# External dependent service DEV URLs
+api:
+  base:
+    url:
+      activities-appointments: https://activities-api-dev.prison.service.justice.gov.uk
+      hmpps-auth: https://sign-in-dev.hmpps.service.justice.gov.uk/auth
+      locations-inside-prison: https://locations-inside-prison-api-dev.hmpps.service.justice.gov.uk
+      manage-users: https://manage-users-api-dev.hmpps.service.justice.gov.uk
+      prison-api: https://prison-api-dev.prison.service.justice.gov.uk
+      prisoner-search: https://prisoner-search-dev.prison.service.justice.gov.uk
+      nomis-mapping: https://nomis-sync-prisoner-mapping-dev.hmpps.service.justice.gov.uk
```

Comment:
This is to run local with localstack isn't it? Rather than any integration tests with localstack.

Reply:
Yes, just for running locally, nothing to do with tests.